src/main/java/de/tum/in/www1/artemis/domain/metis/conversation/Conversation.java

```diff
@@ -58,11 +58,11 @@ public abstract class Conversation extends DomainObject {
     private User creator;
 
     @JsonIgnoreProperties("conversation")
-    @OneToMany(mappedBy = "conversation", cascade = CascadeType.REMOVE, fetch = FetchType.LAZY)
+    @OneToMany(mappedBy = "conversation", cascade = CascadeType.REMOVE, fetch = FetchType.EAGER)
     private Set<ConversationParticipant> conversationParticipants = new HashSet<>();
 
     @JsonIgnoreProperties("conversation")
-    @OneToMany(mappedBy = "conversation", cascade = CascadeType.REMOVE, fetch = FetchType.LAZY)
+    @OneToMany(mappedBy = "conversation", cascade = CascadeType.REMOVE, fetch = FetchType.EAGER)
```

Comment:
There might be quite a lot of posts inside a conversation. Should we really load all of them?